tests/Unit/Execution/Utils/CustomGlobalIdTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Tests\Unit\Execution\Utils;
+
+use Tests\TestCase;
+use Nuwave\Lighthouse\Support\Contracts\GlobalId;
+use Nuwave\Lighthouse\Execution\Utils\GlobalId as BaseGlobalId;
+
+class CustomGlobalIdTest extends TestCase
```

Comment:
I think this class can be removed. All it really does is to ensure the Service Container works, nothing specific about Lighthouse in here.